src/components/RadioBorderedBox/__stories__/index.stories.tsx

```diff
@@ -132,7 +132,7 @@ Badge.decorators = [
   () => (
     <ControlValue value="badge-1">
       {({ value, onChange }) => (
-        <div style={{ display: 'flex', flexDirection: 'column', gap: 8 }}>
+        <div style={{ display: 'flex', flexDirection: 'column', gap: 16 }}>
```

Comment:
should we use theme here?

Reply:
@vincentaudebert doesn't really apply for stories yet